lang/lang.common.json

```diff
@@ -3,7 +3,11 @@
         "name": "My Bot",
         "author": "My Name"
     },
-    "commands": {
+    "contextCommands": {
+        "viewJoinDate": "View join date!",
+        "viewMessageSentDate": "View message sent date!"
```

Comment:
Can we name these as "View Date Joined" and "View Date Sent", similar with the keys "viewDateJoined", "viewDateSent", and anywhere else named.

Reply:
is this in reference to the variable name of the value?

Reply:
oh I see